<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentlyConverter</class>
 <widget class="QWidget" name="CurrentlyConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(126, 126, 126);</string>
  </property>
  <widget class="QPushButton" name="button_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_negative">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+/-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_minus">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 80, 25);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_percent">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ac">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_0">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_addition">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 80, 25);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_comma">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_del">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_equal">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 80, 25);</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_multiplication">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 80, 25);</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_divide">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 80, 25);</string>
   </property>
   <property name="text">
    <string>÷</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_amount">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_result">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_to">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_from">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0); /* Son rakam olan 0, tam şeffaflık sağlar */
border: none;</string>
   </property>
   <property name="text">
    <string>─────────────────────</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_convertIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0); /* Son rakam olan 0, tam şeffaflık sağlar */
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/-/doubleArrow128.png</normaloff>:/-/doubleArrow128.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="button_return">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0); /* Son rakam olan 0, tam şeffaflık sağlar */
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
